# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: c09a51622e98
# Full Hash: c09a51622e988e81077c75e3aaa5a49df66284b3
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-22 16:44:08
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 737807563dd5 -r c09a51622e98 modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Thu Mar 21 23:14:26 2019 +0000
+++ b/modules/libpref/init/all.js	Fri Mar 22 03:15:14 2019 +0000
@@ -974,6 +974,7 @@
 pref("gfx.webrender.dl.dump-parent", false);
 pref("gfx.webrender.dl.dump-content", false);
 pref("gfx.webrender.picture-caching", true);
+pref("gfx.webrender.split-render-roots", false);
 
 #ifdef NIGHTLY_BUILD
 pref("performance.adjust_to_machine", true);